fix: add secret name and new configs

z103cb opened the pull request:
## Description
Corrected secret name and added branch merging configuration

## How Has This Been Tested?
Manualy

## Merge criteria:
- [X] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

z103cb committed:
fix: add secret name and new configs

Corrected secret name and added branch mergeing configuration

z103cb commented on src/config/source_map.yaml:
```diff
@@ -1 +1,83 @@
+    branch: releases/20*
+  dest:
+    url: https://github.com/opendatahub-io/openvino_contrib.git
+    branch: fast
```
@heyselbi, @Xaenalt and @rpancham can you confirm that this is the branch we went to create the new releases in ?

Xaenalt replied:
Sure, that works for me

heyselbi replied:
fast branch works for me too!

z103cb replied:
Cool. Then LGTM when this PR.

heyselbi commented on src/config/source_map.yaml:
```diff
@@ -1 +1,83 @@
+    branch: releases/2023/3
+  dest:
+    url: https://github.com/opendatahub-io/openvino.git
+    branch: 2023.3-release
```
@Xaenalt @z103cb I thought decision was to have a more generic branch name so it could be a "continuous" release? Just wanted to confirm that having a specific release branch name would not impede the "continuity" of the release sync.

z103cb replied:
@heyselbi let me check the diagram.

z103cb replied:
I have changed the names to better match what is the diagram, I have only changed branch name `new` to `new_release`

heyselbi commented on src/config/source_map.yaml:
```diff
@@ -1 +1,83 @@
+  mode: 'release-following'
+  src:
+    url: https://github.com/openvinotoolkit/openvino_contrib.git
+    branch: releases/20*
```
just curious for my own development: the star would select the latest branch?

z103cb replied:
The regex is used [here](https://github.com/opendatahub-io/sync-upstream-repo/blob/09e531088b53467575cf9e9ade6ffe75a1361d9b/entrypoint.sh#L174)

z103cb committed:
fix: job and branch name updates

Updated branches and corresponding  jobs to follow this pattern: current upstream release ->  origin release branch, new upstrem branch -> new_release
